### PR TITLE
[Delivers #101608800] amount limit tied to FY start

### PR DESCRIPTION
**Pull request by pkarman**
Story: https://www.pivotaltracker.com/story/show/101608800

### CODE REVIEW

**Comment by pkarman** [comment]
need to fix tests

**Comment by pkarman** [comment]
tests ok

File: app/models/ncr/work_order.rb [hunk 1]
```diff
@@ -8,6 +8,13 @@ def self.table_name_prefix
 
   EXPENSE_TYPES = %w(BA60 BA61 BA80)
   BUILDING_NUMBERS = YAML.load_file("#{Rails.root}/config/data/ncr/building_numbers.yml")
+  FY16 = Time.zone.parse('2015-10-01')
```
Comment (phirefly) [inline app/models/ncr/work_order.rb]: Should we share, since this is also being used with gsa18f?

Comment (pkarman) [inline app/models/ncr/work_order.rb]: share that one line? `FY16 = ...` ?

the logic for setting MAX_AMOUNT?

or share the whole `validates` block?

if the latter, I think the only way to do that would be with a Concern mix-in. Right now there are 2 different validation messages (the default, and our custom one in `Ncr::WorkOrder`). Do we want to have the same message across both classes?

Comment (afeld) [inline app/models/ncr/work_order.rb]: "Yes" to all of these things. Can call the Concern `PurchaseCardUseCase` or `PurchaseCardMixin` or `PurchaseCardRequest` or something.

File: app/models/concerns/purchase_card_mixin.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+module PurchaseCardMixin
```
Comment (afeld) [inline app/models/concerns/purchase_card_mixin.rb]: I'd add a comment at the top saying "requires `.purchase_amount_column_name` method to be defined before `include`ing".

**Comment by afeld** [comment]
Just one minor comment...otherwise :shipit:

**Comment by pkarman** [comment]
@afeld thanks, commented added.

**Pull request merged**